nix/shell.sh

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env bash
+
+export TERM=xterm # fix for colors
```

Comment:
I'm not entirely sure hardcoding this is a good idea, but we'll see in CI output if it is.

Also, I kinda liked `nix/shell`, adding the `.sh` makes it clunky :D.

Reply:
TBH I only added it because it makes VS Code recognize the file type and do the right syntax highlighting :-)

Reply:
Ah, weird it doesn't recognize it from the shebang.